nix/release.nix

```diff
@@ -0,0 +1,7 @@
+# This file wires up the different vaultenv releases that are available.
+rec {
+  pkgs = import ./nixpkgs-pinned.nix {};
```

Comment:
This doesn't seem like it should be member of the set of released packages. Perhaps it fits better in a let-binding.

Reply:
I moved it to a let binding.